examples/browser/package.json

```diff
@@ -10,6 +10,7 @@
     }
   },
   "dependencies": {
+    "@theia/branding": "file:../../packages/branding",
```

Comment:
why do you use link dependencies, we use yarn workspaces, they already properly link everyting, just `"@theia/branding": "^0.3.18"`

https://yarnpkg.com/lang/en/docs/workspaces/

Reply:
yes sorry it was only temporary, I initially had errors and thought it might be the culprit, I made the updates now to the dependencies.